### PR TITLE
Some fixes to layout matching wireframe

### PR DESCRIPTION
**Pull request by Pooch11**
Presents a working base to start style fixes with client layout.

Details on Button position needed 
Writers box will need an actual rendering element. - placeholder for now.
Sidebar needs to be accounted for - shrinking the screen does not scale this portion.

### CODE REVIEW

File: client/src/components/app.js [hunk 1]
```diff
@@ -12,9 +12,10 @@ const App = Client({
 
 const CreativeCollab = () => (
   <div className="container" style={{ padding: 50 }}>
-    <h1>CreativeColab</h1>
+    <h1>CreativeColab1</h1>
```
Comment (klymenkoo) [inline client/src/components/app.js]: Typo?

**Comment by SeanPrashad** [comment]
Thanks for this patch @Pooch11 - I know its a WIP but you'll have to rebase off of `master` sometime before we can land this patch - please give me a shout if you need any help with that!

File: client/src/components/creative-collab.css [hunk 2]
```diff
@@ -1,12 +1,30 @@
-.editor,
+.editor {
+  width: 95%;
+  height: 200px;
```
Comment (klymenkoo) [inline client/src/components/creative-collab.css]: Try to stick with one unit (either px or percentage). Probably, you would go with % in order to keep it responsive.

File: client/src/components/creative-collab.css [hunk 3]
```diff
@@ -1,12 +1,30 @@
...
-  max-width: 500px;
+  max-width: 1500px;
   max-height: 400px;
 }
-
+#players-container {
+  text-align: left;
+  width: 15%;
+  height: inherit;
+  max-width: 15%;
```
Comment (klymenkoo) [inline client/src/components/creative-collab.css]: Same as above. Try to stick with one unit.

File: client/src/components/app.js (hunk 1 above)
Comment (Pooch11) [reply client/src/components/app.js]: yep - wanted to see if things were changed in real time :)

Comment (SeanPrashad) [reply client/src/components/app.js]: Yep they are! We have [HMR (hot module replacement)](https://webpack.js.org/guides/hot-module-replacement/) which will automatically refresh things as you edit them - pretty neat stuff

**Comment by SeanPrashad** [comment]
#28 has landed into `master` - can you please rebase and fix up the merge conflicts @Pooch11?

`git checkout master && git pull upstream master && git checkout - && git rebase master`

**Pull request merged**